app/Infrastructure/Http/Resources/HolidayResource.php

```diff
@@ -19,6 +19,10 @@ public function toArray($request): array
             "isPast" => $this->date->endOfDay()->isPast(),
             "displayDate" => $this->date->toDisplayString(),
             "dayOfWeek" => $this->date->dayName,
+            "daysToHoliday" => $this->date->diffForHumans([
+                "parts" => 2,
+                "join" => " i ",
+            ]),
```

Comment:
Why not simpler?
```suggestion
            "daysToHoliday" => $this->date->diffForHumans(),
```

Reply:
it is show only weeks or month when holiday will became but its not problem to change that.

Reply:
I think that "in 2 months" in more readable than "in 2 months and 12 days". ;)